pkg/kritis/review/review_test.go

```diff
@@ -121,79 +147,103 @@ func TestReviewGAP(t *testing.T) {
 		image        string
 		policies     []v1beta1.GenericAttestationPolicy
 		attestations []metadata.PGPAttestation
-		isAttested   bool
+		isAdmitted   bool
 		shouldErr    bool
 	}{
 		{
 			name:         "valid image with attestation",
 			image:        img,
 			policies:     oneGAP,
 			attestations: validAtts,
-			isAttested:   true,
+			isAdmitted:   true,
 			shouldErr:    false,
 		},
 		{
 			name:         "image without attestation",
 			image:        img,
 			policies:     oneGAP,
 			attestations: []metadata.PGPAttestation{},
-			isAttested:   false,
+			isAdmitted:   false,
+			shouldErr:    true,
+		},
+		{
+			name:         "gap without attestor should error",
+			image:        img,
+			policies:     gapWithoutAA,
+			attestations: []metadata.PGPAttestation{},
+			isAdmitted:   false,
+			shouldErr:    true,
+		},
+		{
+			name:         "gap without attestor should error on allowlisted image",
+			image:        "allowed_image_name",
+			policies:     gapWithoutAA,
+			attestations: []metadata.PGPAttestation{},
+			isAdmitted:   false,
 			shouldErr:    true,
 		},
 		{
 			name:         "allowlisted image",
 			image:        "allowed_image_name",
 			policies:     oneGAP,
 			attestations: []metadata.PGPAttestation{},
-			isAttested:   false,
+			isAdmitted:   false,
 			shouldErr:    false,
 		},
 		{
 			name:         "image allowlisted in 1 policy",
 			image:        "allowed_image_name",
 			policies:     twoGAPs,
 			attestations: []metadata.PGPAttestation{},
-			isAttested:   false,
+			isAdmitted:   false,
 			shouldErr:    false,
 		},
 		{
 			name:         "image without policies",
 			image:        img,
 			policies:     []v1beta1.GenericAttestationPolicy{},
 			attestations: []metadata.PGPAttestation{},
-			isAttested:   false,
+			isAdmitted:   false,
 			shouldErr:    false,
 		},
 		{
 			name:         "image with invalid attestation",
 			image:        img,
 			policies:     oneGAP,
 			attestations: invalidAtts,
-			isAttested:   false,
+			isAdmitted:   false,
 			shouldErr:    true,
 		},
 		{
 			name:         "image complies with one policy out of two",
 			image:        img,
 			policies:     twoGAPs,
 			attestations: validAtts,
-			isAttested:   true,
+			isAdmitted:   true,
 			shouldErr:    false,
 		},
 		{
 			name:         "image in global allowlist",
 			image:        "us.gcr.io/grafeas/grafeas-server:0.1.0",
 			policies:     twoGAPs,
 			attestations: []metadata.PGPAttestation{},
-			isAttested:   false,
+			isAdmitted:   false,
 			shouldErr:    false,
 		},
 		{
 			name:         "image attested by one attestor out of two",
 			image:        img,
 			policies:     gapWithTwoAAs,
 			attestations: validAtts,
```

Comment:
Can you rename "validAtts" to something more easily distinguishable from "twoValidAtts"?